source/configure/configuration-in-your-database.rst

```diff
@@ -119,17 +115,18 @@ Here's a complete ``mattermost.service`` file with the ``EnvironmentFile`` line
 Migrate configuration from ``config.json``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can use the :ref:`mmctl config migrate <manage/mmctl-command-line-tool:mmctl config migrate>` command to migrate the configuration.
+You can use the :ref:`mmctl config migrate <manage/mmctl-command-line-tool:mmctl config migrate>` command to migrate the configuration by running command:
 
-.. note::
- 
-   If you're using a High Availability cluster, you only need to run this on a single server in the cluster.
+.. code-block: none
 
-.. warning::
-   
-   When migrating config, Mattermost will incorporate configuration from any existing ``MM_*`` environment variables set in the current shell. See :doc:`Environment Variables  </configure/configuration-settings>`
-   
-As with the environment file, you'll have to escape any single quotes in the database connection string. Also, any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster.
+   mmctl config migrate path/to/config.json "postgres://mmuser:mostest@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10" --local
+
+.. important::
+ 
+   - If you're using a High Availability cluster, you only need to run this command on 1 server in the cluster.
+   - When migrating configuration, Mattermost incorporates configuration from any existing ``MM_*`` environment variables set in the current shell. See :doc:`Environment Variables  </configure/configuration-settings>` documentation for details.
+   - As with the environment file, you'll have to escape any single quotes in the database connection string. 
+   - Any existing SAML certificates will be migrated into the database as well so they are available for all servers in the cluster. When the certificates expire, you can upload new certificates using the System Console or mmctl, which triggers a database update. Replacing the certificate files manually requires a reload of the Mattermost server to repull the certificates.
```

Comment:
We should note that files stored for config are stored in the `configurationfiles` table on the database where the config is stored